### ISSUE TITLE
Alternative

### ISSUE THREAD

**Issue by vednolacni** [GimpArm/smartfriends-bridge#8]
Did anybody make this work with Schellenberg SH1-Box or Schellenberg USB (21009) ?

**Comment by MarcCain-Scott** [comment]
I replied to your previous question about the USB stick here https://github.com/GimpArm/smartfriends-bridge/issues/5#issuecomment-2210278439

As for the Schellenberg SH1-Box, I would expect it to work as long as you entered in all the correct setup information although I have no tested because I do not have access to one.

**Comment by GimpArm** [comment]
I replied to your previous question about the USB stick here https://github.com/GimpArm/smartfriends-bridge/issues/5#issuecomment-2210314492

As for the Schellenberg SH1-Box, I would expect it to work as long as you entered in all the correct setup information although I have no tested because I do not have access to one.

**Comment by GimpArm** [comment]
So I got the stick. It was rather easy to get this working. One thing they should have noted in their repo is that the IDs `00` and `01` are not allowed, a little point of frustration for me. Also the receiving message structure is slightly different than what is written, probably changed due to updates. There is no way that I can see to get or set the position but when the blinds move or stop they send a signal so it is possible to time them and make a guess at their position.

I created a new repo for the project and I will be placing my work there. https://github.com/GimpArm/schellenberg-usb
